integration/test/Test/Connection.hs

```diff
@@ -401,3 +402,14 @@ testFederationAllowMixedConnectWithRemote =
     connectTwoUsers alice bob
   where
     defSearchPolicy = "full_search"
+
+testPendingConnectionUserDeleted :: HasCallStack => Domain -> App ()
+testPendingConnectionUserDeleted domain = do
```

Comment:
```suggestion
testPendingConnectionUserDeleted bobsDomain = do
```

?  I sometimes feel we're trying really hard to make identifiers mean as little as possible in /integration.  could be just me, though :)